xbps-src

```diff
@@ -78,6 +78,9 @@ remove <pkgname>
 remove-autodeps
     Removes all package dependencies that were installed automatically.
 
+remove-obsoletes
+    Remove all obsolete packages from default repositories as detected by xbps-rindex.
```

Comment:
it removes from repositories resulting from building in branch or from `xbps-src -r` too.

Reply:
Indeed.  I'm not sure yet that this is a problem, as the primary consumer of this feature is Void itself in an environment where there are no branches.

Reply:
I think it's worth documenting it, at least?

Reply:
```suggestion
    Remove all obsolete packages from $hostdir/binpkgs as detected by xbps-rindex.
```

or something like that.

Reply:
I think what I was more getting at was I'm not convinced that the automatic folder detection was correct.  I wonder if its better to just have a static list.